source/tyr/tests/integration/dataset_test.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import absolute_import, print_function, unicode_literals, division
+from tests.check_utils import api_get
+import pytest
+from navitiacommon import models
+from tyr import app
+
+@pytest.fixture
+def create_basic_job_with_data_sets():
+    with app.app_context():
+        instance = models.Instance('fr')
+        models.db.session.add(instance)
+        models.db.session.commit()
+
+        job = models.Job()
+        job.instance = instance
+
+        # we also create 2 datasets, one for fusio, one for synonym
+        for i, dset_type in enumerate(['fusio', 'synonym']):
+            dataset = models.DataSet()
+            dataset.type = dset_type
+            dataset.family_type = dataset.type
+            if dataset.type == 'fusio':
+                dataset.family_type = 'pt'
+            dataset.name = '/path/to/dataset_{}'.format(i)
+            models.db.session.add(dataset)
+
+            job.data_sets.append(dataset)
+
+        job.state = 'done'
+        models.db.session.add(job)
+        models.db.session.commit()
+
+
+@pytest.fixture
+def create_jobs_with_three_data_sets():
```

Comment:
There is a lot of duplicated code from the test above. Could this be re-factored please ?